packages/graphback-codegen-schema/tests/__snapshots__/GraphQLSchemaCreatorTest.ts.snap

```diff
@@ -285,9 +288,10 @@ type Fix {
   id: ID!
 
   \\"\\"\\"
-  Added as part of fixing https://github.com/aerogear/graphback/issues/1452 showing that even if filtering is disabled on a model level,
+  @oneToMany(field: 'bugFix', key: 'bugFixId') 
+   Added as part of fixing https://github.com/aerogear/graphback/issues/1452 showing that even if filtering is disabled on a model level,
   we should have filter type created as it can be used on the relationship level without throwing an error.
-  @oneToMany(field: 'bugFix', key: 'bugFixId')
+  @oneToMany(field: 'bugFix')
```

Comment:
I'll prefer not to repeat this annotation but It is safe to have this year, as graphback will recognise the relationship annotation in https://github.com/aerogear/graphback/pull/1818/files#diff-3a7edbeb794d4f75d443877f36311dd6R291